src/subapps/search/components/ColumnsVisibilityConfig/index.tsx

```diff
@@ -0,0 +1,100 @@
+import { EyeInvisibleOutlined } from '@ant-design/icons';
+import { Button, Form, Switch } from 'antd';
+import Modal from 'antd/lib/modal/Modal';
+import * as React from 'react';
+import './ColumnsVisibility.less';
+
+export type ColumnVisibility = {
+  key: string;
+  name: string;
+  visible: boolean;
+};
+
+const ColumnsVisibilityConfig: React.FunctionComponent<{
+  columns: ColumnVisibility[];
+  onSetColumnVisibility: (columnVisibility: ColumnVisibility) => void;
+  onSetAllColumnVisibile: () => void;
+}> = props => {
```

Comment:
we can desctructure props here so we don't need to use props.something everywhere:
```
}=> ({columns, onSetColumnVisibility, onSetAllColumnVisibile}) => {
```